main.go

```diff
@@ -25,22 +26,32 @@ import (
 
 func main() {
 	var (
-		lokiUrl   string
-		username  string
-		password  string
-		rawLabels string
+		lokiUrl              string
+		username             string
+		password             string
+		rawLabels            string
+		interruptWaitSeconds int64
 	)
 
 	fs := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
 	fs.StringVar(&lokiUrl, "addr", "", "Server address. Defaults to $LOKI_ADDR if not set.")
 	fs.StringVar(&username, "username", "", "Username for basic auth. Defaults to $LOKI_USERNAME if not set.")
 	fs.StringVar(&password, "password", "", "Password for basic auth. Defaults to $LOKI_PASSWORD if not set.")
 	fs.StringVar(&rawLabels, "labels", `{job="lokitee"}`, `Labels to inject for logs. i.e., {app="shell"}`)
+	fs.Int64Var(&interruptWaitSeconds, "interrupt-wait", 0, "Number of seconds to delay exiting if sending an interrupt signal like SIGTERM is sent")
```

Comment:
Can you use time.Duration instead of int64 and use `fs.DurationVar` instead? That way you could set a sleep for somewhere between >0s and <1s (ie `250ms`)

Reply:
I didn't know about this! will do